image-fetcher/fetch-image.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -ex
+
+gcloud auth activate-service-account \
+  --key-file=/secrets/gcr-pull.json
+
+gcloud auth configure-docker
+
+while [ 1 ]
+do
+    [ $(cat images | wc -l) -eq 0 ] || (cat images | xargs docker pull)
```

Comment:
The first half seems redundant, xargs won't call docker if there are no lines in images.

Reply:
Yeah, I thought so too. In fact, it works this way on OS X, but not on alpine, which `docker:dind` is based on, AFAICT. This definitely works in the cluster though (I have it running right now).

Reply:
(the file is definitely empty too, I checked that)